octodns/provider/route53.py

```diff
@@ -459,7 +459,7 @@ def populate(self, zone, target=False, lenient=False):
                 record_name = zone.hostname_from_fqdn(rrset['Name'])
                 record_name = _octal_replace(record_name)
                 record_type = rrset['Type']
-                if record_type == 'SOA':
+                if record_type not in self.SUPPORTS:
```

Comment:
Does it make sense to try and detect the Route53 alias records here to and ignore them?

Reply:
Sorry, yeah it does. I wanted to give the original PR submitter a chance to do it first, since he already did that work in #27. I'll update this if he doesn't get a chance to.

Reply:
Checking back in on this. Would be good to get this cleared up and tested.

Reply:
@michaelmcallister, are you still interested/do you have time to take a look at adding your fix from #27 here?

Please let me know, I'll probably have a look at incorporating your stuff towards the end of the week if you don't get a chance to, so this PR can be closed. I think I can add you as a co-author on those commits so you would still get credit.

Reply:
I'll work on it now @mvermaes  - sorry for the delay!

Reply:
Done see: https://github.com/github/octodns/pull/216

Reply:
Great, thanks for that!

@ross let us know if you're OK to merge these two PRs as-is, otherwise I don't mind if @michaelmcallister incorporates my trivial edit into #216.